features/find_car.feature

```diff
@@ -0,0 +1,256 @@
+Feature: Find car
+  As a user, I want to use search filters, so I can have quick and easy way to find a car that suits me.
+
+  Background:
+    Given search page is opened
+
+  Scenario: Checking correct postcode
+    When user fill in Postcode field with E203EL
+    And click the Search cars button
+    Then on the page with found cars a Make filter is selected as Any
+    And a Model filter is selected as Any
+    And a Model Variant filter is selected as Any
+    And a Price filter is selected as Any
+    And an Year filter is selected as Any
+    And a Mileage filter is selected as Any
+    And a Body type filter is selected as Any
+    And a Fuel type filter is selected as Any
+    And an Engine size filter is selected as Any
+    And a Fuel consumption filter is selected as Any
+    And an Acceleration filter is selected as Any
+    And a Gearbox filter is selected as Any
+    And a Drivetrain filter is selected as Any
+    And a Doors filter is selected as Any
+    And a Seats filter is selected as Any
+    And a Insurance Group filter is selected as Any
+    And a Annual tax filter is selected as Any
+    And a Colour filter is selected as Any
+    And a Private & trade filter is selected as Any
+    And filter CO2 emissions is selected as Any too
+
+  Scenario: Checking incorrect postcode
+    When user fill in Postcode field with E203E1
+    And click the Search cars button
+    Then message 'Please enter a valid UK postcode' is shown
+
+  Scenario Outline: Checking 'Distance' filter
```

Comment:
As I mentioned in previous comments, there is no need to use `Scenario Outline` if there is only one row in table

Reply:
I understand this. In this scenario and others with one row in a table I use only one row to speed up tests implementation. And I can add more rows for more test if it's needed.